MODEXPS-173: folio-spring-base 5.0.2, Spring Boot 2.7.6

julianladisch opened the pull request.

julianladisch committed:
MDEXP-553: folio-spring-base 5.0.2

alekGbuz commented on pom.xml:
```diff
@@ -46,7 +46,7 @@
 
-		<folio-spring-base.version>3.0.3</folio-spring-base.version>
+		<folio-spring-base.version>5.0.2</folio-spring-base.version>
 		<lombok.version>1.18.18</lombok.version>
```
Folio spring base 5.0.2 has 2.7.3 version  of spring-boot-starter-parent, so it better to use here the same at pom.xml

julianladisch replied:
Good catch, thanks!

julianladisch committed:
MODEXPS-173: Spring Boot 2.7.6

Upgrade Spring Boot from 2.6.6 to 2.7.6 to match the
2.7.x version in folio-spring-base.

This allows to remove manual lombok and log4j2 version entries.

This upgrades lombok from 1.18.18 to 1.18.24
and keeps log4j2 at 2.17.2.
